core/src/main/java/nl/nn/adapterframework/util/flow/V8FlowGenerator.java

```diff
@@ -31,13 +31,13 @@
 public class V8FlowGenerator implements IFlowGenerator {
 	private static Logger log = LogUtil.getLogger(V8FlowGenerator.class);
 
-	private GraphvizEngine engine;
+	private ThreadLocal<GraphvizEngine> engines = new ThreadLocal<>();
```

Comment:
Waarom zit dit in een ThreadLocal?

Reply:
Omdat dat de stomme v8 engine van google threadsafe maakt...